app/views/alchemy/admin/pictures/index.html.erb

```diff
@@ -67,15 +67,14 @@
   </div>
 
   <%= render 'alchemy/admin/partials/search_form',
-    additional_params: [:filter, :tagged_with, :size] %>
+    additional_params: [:size] %>
 <% end %>
 
 <div id="picture_archive" class="resources-table-wrapper with_tag_filter">
   <div class="resources-header">
     <h2>
       <%= @pictures.total_count %>
       <%= Alchemy::Picture.model_name.human(count: @pictures.total_count) %>
-      <%= Alchemy.t("picture_library.filter.#{search_filter_params[:filter]}") if search_filter_params[:filter].present? %>
```

Comment:
I'll wrap my head around it a bit and see if I find a way :) I agree it's useful.

Reply:
@tvdeyen since we can now apply multiple filters, I suggest we render a list of active filters (opposed to building a sentence). We could do that for Attachment and maybe other models as well.

I put this together quickly, what do you think?

<img width="923" alt="Screenshot 2021-07-08 at 14 53 22" src="https://user-images.githubusercontent.com/342977/124925842-28d99480-dffd-11eb-8183-90055e8580ba.png">

Reply:
I changed the styling slightly so the filter list items aren't bold (I prefer that)

<img width="720" alt="Screenshot 2021-07-08 at 15 58 47" src="https://user-images.githubusercontent.com/342977/124935240-c46f0300-e005-11eb-81b6-90a143933fc4.png">

Reply:
Nice! I like that.

Reply:
@tvdeyen cool, I just pushed a commit with the implementation.

I changed the view slightly as I think the sentence was a bit misleading as it reads like "the 1 Picture is filtered by ..." which is not true. The collection is filtered, resulting in 1 Picture.

It looks like this now

<img width="721" alt="Screenshot 2021-07-09 at 10 56 43" src="https://user-images.githubusercontent.com/342977/125052806-a6a4ab00-e0a4-11eb-8cbf-6574513abe06.png">

Do you agree this is better?

It will kick in for most applications using resources as long as they don't customise the resource views.

Reply:
I like that even better, thanks. 

I imagine that people would like to click on one of these filters to remove it? Do you think we can make it possible to dismiss these Filters as well?

![125052806-a6a4ab00-e0a4-11eb-8cbf-6574513abe06](https://user-images.githubusercontent.com/42868/125054655-bde49800-e0a6-11eb-8c04-4cdcb21facff.png)

Reply:
hehe, we are very alike when it comes to those kind of things 😆 I had the same thought and left it out only because I didn't want to add too many things to the already big PR.

I'll have a look at it quickly. It shouldn't be a big change.

Reply:
I know. Thanks. Again, its not mandatory for this PR. Skip it if you like.

Reply:
Challenge already accepted ;)

I pushed another commit.

Reply:
and this is how it looks like now

<img width="721" alt="Screenshot 2021-07-09 at 14 29 57" src="https://user-images.githubusercontent.com/342977/125078237-681de900-e0c2-11eb-831d-aa0e375d5688.png">

Reply:
Many thanks! This is great